django_seo_js/tests/test_templatetags.py

```diff
@@ -1,11 +0,0 @@
-from django.test import TestCase
-from django.template import Template, Context
-
-
-class TagsTest(TestCase):
-
-    def test_seo_js_head(self):
-        t = Template("{% load django_seo_js %}{% seo_js_head %}")
-        c = Context({})
-        rendered_html = t.render(c)
-        self.assertEqual(rendered_html, """<meta name="fragment" content="!">""")
```

Comment:
Not 100% sure. I think it has something to do with `{% load %}` in 1.9. anyway, i remove this test because i know for sure that i am not using this part of the code.